posthog/api/utils.py

```diff
@@ -126,12 +127,20 @@ def get_project_id(data, request) -> Optional[int]:
     return None
 
 
+@limits(calls=10, period=1)
+def capture_as_common_error_to_sentry(error):
+    try:
+        raise Exception("Invalid payload") from error
```

Comment:
Definitely pass along the `error` here so that we have some context from the sample

Reply:
if we raise this from error, then we do get that in the stacktrace, no?

Reply:
oh nm, this is a great exception chain! IGNORE ME